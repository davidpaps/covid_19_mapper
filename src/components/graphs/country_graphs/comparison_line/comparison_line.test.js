import * as React from "react";
import { shallow } from "enzyme";
import ComparisonLineContainer from "./comparison_line";

describe("Comparison Line", () => {
  let wrapper;

  beforeEach(
    () =>
      (wrapper = shallow(
        <ComparisonLineContainer
          data={[]}
          top10Data={[]}
          lineLabels={[]}
          selected={""}
        />
      ))
  );

  it("should render a <div />", () => {
    expect(wrapper.find("div").length).toEqual(5);
  });

  it("should render a <Line />", () => {
    expect(wrapper.find("Line").length).toEqual(1);
  });

  it("should render a <Button />", () => {
    expect(wrapper.find("Button").length).toEqual(2);
  });

  it("renders correctly", () => {
    expect(shallow).toMatchSnapshot();
  });
});
