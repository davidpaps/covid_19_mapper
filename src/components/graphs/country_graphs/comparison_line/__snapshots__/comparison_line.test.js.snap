// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comparison Line renders correctly 1`] = `[Function]`;
